cli/src/lib.rs

```diff
@@ -526,6 +529,45 @@ pub enum ClusterCommand {
     List,
 }
 
+/// Package manager to use for the project.
+#[derive(Clone, Debug, Default, Eq, PartialEq, Parser, ValueEnum, Serialize, Deserialize)]
+pub enum PackageManager {
+    /// Use npm as the package manager.
+    #[default]
+    NPM,
```

Comment:
One concern I have about this is that defaulting to `npm` might cause problems for existing projects given `yarn` has always been the default. It would be great to avoid last minute breaking changes since we'll have the next version out soon™ (it was supposed to be out last month, but got delayed unfortunately).

We can switch the default to `npm` in v0.32, which will (most likely) include a big refactor of `Anchor.toml`.

Reply:
Alright, makes sense